From 4f26b2252e90b27ff3c0a5287735dfae8dfa86a2 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Mon, 1 Sep 2008 09:57:25 +1000
Subject: [PATCH 27/38] exa: drop cw.h inclusion

this doesn't seem to be needed, at least I can't see any warning without it.

I'd like to build EXA into a driver and cw.h isn't exported
(cherry picked from commit 0466b8811ce8ebc1d993f177cbb7a22e997640f1)
---
 exa/exa.c           |    1 -
 exa/exa_accel.c     |    1 -
 exa/exa_migration.c |    1 -
 3 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/exa/exa.c b/exa/exa.c
index b642bbd..c8e345a 100644
--- a/exa/exa.c
+++ b/exa/exa.c
@@ -36,7 +36,6 @@
 
 #include "exa_priv.h"
 #include "exa.h"
-#include "cw.h"
 
 DevPrivateKey exaScreenPrivateKey = &exaScreenPrivateKey;
 DevPrivateKey exaPixmapPrivateKey = &exaPixmapPrivateKey;
diff --git a/exa/exa_accel.c b/exa/exa_accel.c
index 1c07a0b..522224b 100644
--- a/exa/exa_accel.c
+++ b/exa/exa_accel.c
@@ -34,7 +34,6 @@
 #include <X11/fonts/fontstruct.h>
 #include "dixfontstr.h"
 #include "exa.h"
-#include "cw.h"
 
 static void
 exaFillSpans(DrawablePtr pDrawable, GCPtr pGC, int n,
diff --git a/exa/exa_migration.c b/exa/exa_migration.c
index c5c47df..571650c 100644
--- a/exa/exa_migration.c
+++ b/exa/exa_migration.c
@@ -34,7 +34,6 @@
 
 #include "exa_priv.h"
 #include "exa.h"
-#include "cw.h"
 
 #if DEBUG_MIGRATE
 #define DBG_MIGRATE(a) ErrorF a
-- 
1.6.0.3

